holoviews/core/operation.py

```diff
@@ -182,7 +182,7 @@ def _process(self, view, key=None):
         raise NotImplementedError
 
 
-    def process_element(self, element, key, **params):
+    def process_element(self, element, key=None, **params):
```

Comment:
Why is ``key`` now optional? From memory, the purpose of this method was to always supply a key and get the corresponding result...